example/src/pages/Dashboard/Transactions/index.tsx

```diff
@@ -22,14 +30,17 @@ const Transactions = () => {
       timeout: 3000,
       contractAddress
     }).then(({ data, success }) => {
+      refreshAccount();
       setState({
         transactions: data,
         transactionsFetched: success
       });
     });
   };
 
-  React.useEffect(fetchData, []);
+  React.useEffect(fetchData, [
+    successfulTransactions.successfulTransactionsArray.length
```

Comment:
useGetSuccessfulTransactions also exposes a boolean "hasSuccesfulTransactions", use that